VideoLocker/src/main/java/org/edx/mobile/view/MyVideosActivity.java

```diff
@@ -2,19 +2,43 @@
 
 import android.os.Bundle;
 import android.support.design.widget.TabLayout;
-import android.support.v4.view.PagerAdapter;
 import android.support.v4.view.ViewPager;
 import android.view.View;
 import android.view.WindowManager;
 
 import org.edx.mobile.R;
 import org.edx.mobile.base.BaseVideosDownloadStateActivity;
+import org.edx.mobile.logger.Logger;
 import org.edx.mobile.module.analytics.ISegment;
 import org.edx.mobile.view.adapters.StaticFragmentPagerAdapter;
 
-public class MyVideosActivity extends BaseVideosDownloadStateActivity {
+public class MyVideosActivity extends BaseVideosDownloadStateActivity implements MyRecentVideosFragment.MyVideosTabListener {
 
     private View offlineBar;
+    private final Logger logger = new Logger(MyVideosActivity.class.getName());
+    StaticFragmentPagerAdapter adapter;
+
+    public void refreshMyAllVideosTab() {
+        // TODO Pick one of these
```

Comment:
@mdinino @1zaman Note to reviewers, when a video is deleted from "Recent Videos" the "All Videos" tab is not updated. In this blurb of code, we have a callback that gets the fragment from the StaticFragmentPagerAdapter adapter and calls the refresh. I wrote two different ways (both I'm not too ecstatic about) to get the correct fragment. What are you thoughts?